ui_framework/doc_site/src/views/app_view.jsx

```diff
@@ -35,9 +33,9 @@ export default class AppView extends Component {
   getChildContext() {
     return {
       openCodeViewer: this.props.openCodeViewer,
-      updateCodeViewer: this.props.updateCodeViewer,
-      registerCode: this.props.registerCode,
-      unregisterCode: this.props.unregisterCode,
+      registerSection: this.props.registerSection,
+      unregisterSection: this.props.unregisterSection,
+      sections: this.props.sections,
```

Comment:
On that context note: all of these should be removed from context and rather fetched from the redux store in containers where they are needed (so either pass them through as props all the way down, or insert containers in the tree that fetches it from the store).